listeners/measured.go

```diff
@@ -27,14 +27,14 @@ func (l *stateAwareMeasuredListener) Accept() (c net.Conn, err error) {
 	sac, _ := c.(*measured.Conn).Conn.(WrapConnEmbeddable)
 	return &wrapMeasuredConn{
 		WrapConnEmbeddable: sac,
-		Conn:               *c.(*measured.Conn),
+		Conn:               c.(*measured.Conn),
```

Comment:
Except for idleConn, all those other are okay because they don't involve any copying.  I fixed idleConn by making it embed net.Conn instead of idletiming.Conn.